src/content/developers/docs/apis/json-rpc/reference/index.md

```diff
@@ -5,7 +5,11 @@ lang: en
 sidebar: true
 ---
 
-## Gossip Methods
+A handful of core JSON-RPC endpoints require data from the Ethereum network, and fall neatly into three main categories: [Gossip](#gossip_methods), [State](#state_methods), and [History](#history_methods)
```

Comment:
This opener should reference "Methods" if possible

Reply:
Rethought this.. there was really no need to use that term. Switched to using the term "endpoint" and added this to a pre-req section on the overview page.